backend/src/main/java/com/google/sps/datastore/MatchDatastore.java

```diff
@@ -40,41 +41,39 @@ public long addMatch(Match match) {
     // Insert entity into datastore
     datastore.put(matchEntity);
 
-    // Return matchKey
+    // Return match key id
     return matchEntity.getKey().getId();
   }
 
-  /** Return match based on match datastore key id */
-  @Nullable
-  public Match getMatchFromId(long matchId) {
-    Key matchKey = KeyFactory.createKey(KIND_MATCH, matchId);
-    try {
-      return getMatchFromEntity(datastore.get(matchKey));
-    } catch (EntityNotFoundException e) {
+  /** Return Match from entity, or null if entity is null */
+  private Match getMatchFromEntity(Entity entity) {
```

Comment:
tiny nit: I like having a method like this to convert from the datastore entity to the Plain Old Java Object (POJO) representation. Can it be static?

Reply:
Done.